packages/pulp-puppet/pulp-puppet.spec

```diff
@@ -13,13 +13,13 @@
 
 # ---- archive related macros ----
 
-%define git_tag %{version}
+%define git_tag 2.18-release
```

Comment:
should this point to a tag instead of a branch?

Reply:
see above comment.